io.jamulus.Jamulus.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: io.jamulus.Jamulus
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: Jamulus
+rename-icon: jamulus
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--share=network'
+  - '--filesystem=xdg-run/pipewire-0'
```

Comment:
Running under --log-session bus I see that messages to accessibility are being filtered. You may need: '--talk-name=org.a11y.Bus'